tap-client/src/components/home/HomeComponent.tsx

```diff
@@ -1,8 +1,107 @@
+import collegeImg from "../../../src/assets/home-assets/stock-TU.jpg"
+import capLottie from "../../../src/assets/home-assets/capLottie.json"
+import aiLottie from "../../../src/assets/home-assets/aiLottie.json"
+import studentsLottie from "../../../src/assets/home-assets/studentsLottie.json"
+import Lottie from "lottie-react";
```

Comment:
Test some other lottie images out because the cap and students don't animate as frequently.

Reply:
So you want me to swap out the cap and students for other Lottie images that animate more frequently or you want me to swap the ai lottie for something that animates less frequently?